framework/db/Schema.php

```diff
@@ -281,6 +281,23 @@ public function refresh()
     }
 
     /**
+     * Refreshes the particular table schema.
+     * This method cleans up cached table schema so that it can be re-created later
+     * to reflect the database schema change.
+     * @param string $name table name.
+     */
```

Comment:
`@since` tag added.

> I also suggest exposing this method in Connection so that it is more visible and easier to use.

Since we do not expose `Schema::refresh()` at `Connection` level, we should not move `Schema::refreshTableSchema()` there as well.
Currently it is easy enough to access those methods:
```php
Yii::$app->db->schema->refresh();
Yii::$app->db->schema->refreshTableSchema($tableName);
```